include/NovelRT/Animation/SpriteAnimatorFrame.h

```diff
@@ -27,12 +27,12 @@ namespace NovelRT::Animation {
       _texture = value;
     }
 
-    inline float getDuration() const noexcept {
+    inline const float& duration() const noexcept {
```

Comment:
Do we have an issue tracking changing this to `Timestamp`?

Reply:
No but it is on my to do from the review and will be doing it in a separate PR. 😄